examples/benchmarks/rpl-req-resp/Makefile

```diff
@@ -1,8 +1,9 @@
 CONTIKI_PROJECT = node
 all: $(CONTIKI_PROJECT)
 
-PLATFORMS_EXCLUDE = sky z1 nrf52dk native simplelink
-BOARDS_EXCLUDE = srf06/cc13x0 launchpad/cc1310 launchpad/cc1350 sensortag/cc2650 sensortag/cc1350
+PLATFORMS_EXCLUDE = sky z1 nrf52dk native
+BOARDS_EXCLUDE = srf06/cc13x0 srf06/cc26x0 launchpad/cc1310 \
+	launchpad/cc1350 launchpad/cc1350-4 sensortag/cc2650 sensortag/cc1350
```

Comment:
Can you add `launchpad/cc2650` to `BOARDS_EXCLUDE` - it can also not fit this example

Reply:
We should keep `launchpad/cc2650` because as you say in the your other comment, it can built on cc26x0-cc13x0 with this board. There is no way how to limits specific TARGET/BOARD combination. The situation we have with simplelink and cc26x0-cc13x0 is not something this build exclusion system is prepared for.

Reply:
> it can built on cc26x0-cc13x0 with this board

Just a note: This is true for the others boards as well, you thinking to add one to give a bit more of options to the platform? (There is only the `launchpad/cc2640r2` which is unique to cc26x0-cc13x0).